bciers/apps/reporting/src/app/components/changeReview/templates/ReviewChanges.tsx

```diff
@@ -24,26 +24,29 @@ import {
   reviewChangesNote,
 } from "@reporting/src/data/jsonSchema/changeReview/complianceNote";
 import { ReportingFlow } from "@reporting/src/app/components/taskList/types";
+import {
+  OPTED_IN_OPERATION,
+  REGULATED_OPERATION_REGISTRATION_PURPOSE,
+} from "@reporting/src/app/utils/constants";
 
 export const ReviewChanges: React.FC<ReviewChangesProps> = ({
   changes,
   flow,
+  registrationPurpose,
 }) => {
   // Normalize keys for all changes at the root level
   const normalizedChanges = normalizeChangeKeys(filterExcludedFields(changes));
-  const showChangesFlows = [
-    ReportingFlow.SFO,
-    ReportingFlow.LFO,
-    ReportingFlow.OptedInSFO,
-    ReportingFlow.OptedInLFO,
+  const showChangesPurposes = [
+    REGULATED_OPERATION_REGISTRATION_PURPOSE,
+    OPTED_IN_OPERATION,
   ];
+  const showChanges = showChangesPurposes.includes(registrationPurpose);
 
   const reportingOnlyFlows = [
     ReportingFlow.ReportingOnlySFO,
     ReportingFlow.ReportingOnlyLFO,
   ];
```

Comment:
Maybe for consistency the reporting only can be derived from the registration purpose too